### DISCUSSION TITLE
bottom border for active md-tabs__item

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [squidfunk/mkdocs-material#5048] [Q&A]
Is it possible to customise the **active** md-tabs__item using css, for example adding a bottom border similar to https://design-system.service.gov.uk/get-started/?

I tried the following but it adds a bottom border to all tabs_item:

```
.md-tabs__item {
  border-bottom: 4px solid;
  border-bottom-color:   #1d70b8;
}
```

**Comment by squidfunk** [comment]
#5025 explains [how to target the active tab](https://github.com/squidfunk/mkdocs-material/discussions/5027#discussioncomment-4956046).

**Comment by SoumayaMauthoorMOJ** [comment]
Hey @squidfunk  thanks for the quick response! I did see that answer but it shows how to get the active link not the active item:

```
.md-tabs__item {
  border-bottom: 4px solid;
  border-bottom-color:   #1d70b8;
}
```
![image](https://user-images.githubusercontent.com/95432492/218827886-77d22892-f244-4bec-8aa2-7c9b4a7acea1.png)

```
.md-tabs__link--active {
  border-bottom: 4px solid;
  border-bottom-color:   #1d70b8;
}
```
![image](https://user-images.githubusercontent.com/95432492/218828063-8e2885b3-c5f4-4762-9835-3899e849a2ca.png)

**Comment by SoumayaMauthoorMOJ** [comment]
Would it be possible to modify the tab-item partial so that it has an `md-tab__item--active` class like for `md-nav__item--active`:

![image](https://user-images.githubusercontent.com/95432492/218834821-1a28b8b7-5368-4644-aa8a-a6aaf5e7432b.png)

![image](https://user-images.githubusercontent.com/95432492/218835213-8f1a3709-9c87-4cc6-8774-6586fe80934a.png)

Happy to raise this as an issue or pull request if you prefer :-)

**Reply by squidfunk** [reply]
~~We already have so many CSS classes – I'm not keen on adding even more, since this increases the size of the built site. If you feel that you need the class, please override the `tabs-item` partial and add it for your installation.~~

Or wait, why not. We already have it for navigation as well. You can create a PR.

**Reply by SoumayaMauthoorMOJ** [reply]
Amazing ! I wrote the updated [partial](https://github.com/ministryofjustice/mkdocs-tech-docs-template/blob/v0.0.17/tech_docs_template/partials/tabs-item.html) in my repo for now and will raise PR this weekend :-)
